Taller152/src/paquetetres/EscribirArchivo.java

```diff
@@ -4,11 +4,26 @@
  * and open the template in the editor.
  */
 package paquetetres;
-
+import java.util.Formatter;
 /**
  *
  * @author reroes
  */
 public class EscribirArchivo {
     
+   public static void agregarLibros(String valor) {
+        try {
+            Formatter salida = new Formatter("data/informacion3.txt");
+            salida.format("%s\n", valor);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.